lib/improver/tests/nbhood/circular_kernel/test_CircularNeighbourhood.py

```diff
@@ -396,10 +237,7 @@ def test_single_point_adjacent_edge(self):
         self.assertArrayAlmostEqual(result.data, expected)
 
     def test_single_point_on_edge(self):
-        """Test behaviour for a non-zero grid cell on the edge.
-        Note that this behaviour is 'wrong' and is a result of
-        scipy.ndimage.correlate 'nearest' mode. We need to fix
-        this in the future."""
+        """Test behaviour for a non-zero grid cell on the edge."""
```

Comment:
I'll look into this.

Reply:
This comment has been removed because I didn't think that it was a good idea to have comments in the code describing something as "wrong" without this being captured elsewhere. #266 was created to capture the issue that this comment was raising.

Reply:
We could still have a TODO comment to highlight it, as these test document expected behaviour, although I'm glad we have a ticket to fix it.